_1327/polls/tests.py

```diff
@@ -341,17 +379,17 @@ def test_vote_multiple_choice_correctly(self):
 			data.append(('choice', choice.id))
 			votes.append(choice.votes)
 
-		response = self.app.post(reverse('polls:vote', args=[self.poll.id]), params=data, user=self.user)
+		response = self.app.post(reverse('polls:vote', args=[self.poll.url_title]), params=data, user=self.user)
 		self.assertEqual(response.status_code, 302)
 		for i, choice in enumerate(self.poll.choices.all()):
 			self.assertEqual(choice.votes, votes[i] + 1)
 
 		self.assertEqual(self.poll.participants.count(), 1)
-		self.assertRedirects(response, reverse('polls:results', args=[self.poll.id]))
+		self.assertRedirects(response, reverse('polls:results', args=[self.poll.url_title]))
 
 	def test_view_before_poll_has_started(self):
 		self.poll.start_date += datetime.timedelta(weeks=1)
 		self.poll.save()
 
-		response = self.app.get(reverse('polls:vote', args=[self.poll.id]), user=self.user, expect_errors=True)
+		response = self.app.get(reverse('polls:vote', args=[self.poll.url_title]), user=self.user, expect_errors=True)
 		self.assertEqual(response.status_code, 404)
```

Comment:
yeah right! I actually think that the `polls:vote` and `polls:view` views should not be there anymore I'd like to refactor this when this is done. For now everything works fine, but having just one view that decides on its own which view to show to the user is better I think.